parcels/particleset/particlesetsoa.py

```diff
@@ -427,6 +478,38 @@ def to_dict(self, pfile, time, deleted_only=False):
         return self._collection.toDictionary(pfile=pfile, time=time,
                                              deleted_only=deleted_only)
 
+    def compute_neighbor_tree(self, time, dt):
+        active_mask = self.active_particles_mask(time, dt)
+
+        self._values = np.vstack((
+            self._collection.data['lat'],
+            self._collection.data['lon'],
+            self._collection.data['depth'],
```

Comment:
Again, for consistency throughout Parcels I'd go for depth, lat, lon ordering

Reply:
Done